chat-ui/src/components/Chat/MuiTextInput.tsx

```diff
@@ -78,27 +78,33 @@ export function MuiTextInput({
                 onKeyDown={handleKeyDown}
                 variant="outline"
                 maxH="10rem"
+                bg="gray.200"
+                color="black"
+                shadow="xl"
+                
             />
             <Button
                 type="button"
                 onClick={setResponse}
                 disabled={!value}
                 variant="solid"
-                color="primary"
+                color="gray.700"
                 leftIcon={<AiOutlineSend/>}
+                shadow="xl"
+                bg="gray.200"
             >
                 {sendButtonText}
             </Button>
         </Box>
     );
 }
 function extractContentBetweenBraces(inputString) {
-    const regex = /{{{(.*?)}}}/g;
```

Comment:
This is supposed to be 3 `}` . all the changes in this function should be reverted